src/DocumentFormat.OpenXml/OpenXmlPartReader.cs

```diff
@@ -54,7 +54,7 @@ public OpenXmlPartReader(OpenXmlPart openXmlPart) : this()
                 throw new ArgumentNullException(nameof(openXmlPart));
             }
 
-            _xmlReader = CreateReader(openXmlPart.GetStream(FileMode.Open), true, openXmlPart.MaxCharactersInPart, out _standalone, out _encoding);
+            _xmlReader = CreateReader(openXmlPart.GetStream(FileMode.Open), true, openXmlPart.MaxCharactersInPart, ignoreWhitespace: false, out _standalone, out _encoding);
```

Comment:
Previously, the default value was true. Why are these method, which presumably will have the old behavior, passing in false here?

Reply:
My mistake changed to true